recipes/libudev-devel-cos6-x86_64/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+RPM=$(find ${PWD}/binary -name "*.rpm")
+mkdir -p ${PREFIX}/x86_64-conda_cos6-linux-gnu/sysroot/usr
+pushd ${PREFIX}/x86_64-conda_cos6-linux-gnu/sysroot/usr > /dev/null 2>&1
```

Comment:
May need to drop the `/usr` bit in both of these lines. Depends on how the RPM is installed normally.